### ISSUE TITLE
Files background image relevancy

### ISSUE THREAD

**Issue by yerbestpal** [vinceliuice/Canta-theme#5]
Though I am fond of it, I think the picture in the background of Files, that overlaps the window borders should be changed to something more relevant. Don't get me wrong; I think it looks good and it's something I've always wanted to see, but the current image is just a couple of miscellaneous shapes that actually get in the way of visibility. Maybe something that was more subdued and more relevant to the app e.g. file and folder icons, or an OS logo. Otherwise, great work!

**Comment by vinceliuice** [comment]
Thanks for you advices, This is just a try. I simply want to add a background image with the theme to the file manager. If I find a more suitable image, I will consider replacing it.

**Comment by zesterer** [comment]
I agree with the OP: The image does not at all fit well with content in the file manager. Example:
![image](https://user-images.githubusercontent.com/3268430/37276010-89680d5a-25d8-11e8-8a02-8b307f20f696.png)

**Comment by simonbcn** [comment]
I agree that the background image in Nautilus sometimes hinders the content shown and does not really contribute anything. The decision to have the main installer (`install.sh`) not install that image seems very accurate to me.
